charts/line-chart/src/LineChart.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { Chart, Line } from '@lg-charts/core';
+
+import { LineChartProps } from './LineChart.types';
+
+export function LineChart({ data }: LineChartProps) {
+  return (
+    <Chart>
```

Comment:
Forgot to allow this to be re-styled/sized by the consumer. Will update this shortly

Reply:
Done